java/ruleXtract/src/test/scala/uk/ac/cam/eng/extraction/ExtractSpec.scala

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use these files except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+
+package uk.ac.cam.eng.extraction
+
+import org.scalatest._
+
+class ExtractSpec extends FlatSpec with Matchers {
```

Comment:
Please document the test. The class name should be more meaningful. RuleExtractTest?